css/order.css

```diff
@@ -398,3 +364,191 @@ svg {
   text-align: center;
   transition: 300ms all ease-in-out;
 }
+
+#bnomics-address-input,
+#bnomics-amount-input {
+  padding: 5px;
+  max-width: 190%;
+  color: currentColor;
+  margin: 0;
+}
+
+.bnomics-order-address {
+  align-items: center;
+  max-width: 580px;
+  margin-left: 8px;
+  margin-right: 8px;
+  text-align: left;
+}
+
+.bnomics-or-divider {
+  position: relative;
+  text-align: center;
+  vertical-align: middle;
+  font-size: var(--global--font-size-xs);
+  font-weight: 700;
+  opacity: 0.5;
+
+}
+
+.bnomics-or-divider::before,
+.bnomics-or-divider::after {
+  content: '';
+  display: block;
+  width: 1px;
+  background: currentColor;
+  height: 36%;
+  position: absolute;
+  left: 50%;
+  transform: translateX(-50%);
+}
+
+.bnomics-or-divider::before {
+  top: 16px;
+}
+
+.bnomics-or-divider::after {
+  bottom: 16px;
+}
+
+
+
+.bnomics-order-panel {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  
+  padding-left: 1rem;
+  padding-right: 1rem;
+}
+
+.bnomics-header-container,
```

Comment:
These CSS changes appear to be unnecessary and have little to no affect. Let's ensure we are not including CSS changes that are not required.
We also should not be hardcoding font-size's. We can consider using a relative font size, although this does not appear necessary.